src/core.jl

```diff
@@ -12,9 +12,19 @@ if VERSION < v"0.4.0-dev+4103"
     reverse(p::Pair) = Pair(p.second, p.first)
 end
 
+if VERSION < v"0.4.0-dev"
+    try
+        import Docile: @doc_str
+    catch
+        macro doc_str(x) x end
```

Comment:
This is a little hacky, to get the `doc".."` (required only for LaTeX in Markdown).